packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -749,6 +749,27 @@ export default class TabAdvanced extends Component {
                     />
                   </td>
                 </tr>
+                <tr>
+                  <th>{_('protectFromShutdown')}</th>
+                  <td>
+                    <Toggle
+                      value={'shutdown' in vm.blockedOperations}
+                      onChange={blockStop =>
+                        editVm(vm, {
+                          blockedOperations: {
+                            shutdown: blockStop,
+                            pause: blockStop,
+                            clean_reboot: blockStop,
+                            clean_shutdown: blockStop,
+                            hard_shutdown: blockStop,
+                            hard_reboot: blockStop,
+                            suspend: blockStop,
```

Comment:
Please sort these lines :slightly_smiling_face:

Reply:
Resolved, thank you